R/reporter-progress.R

```diff
@@ -78,15 +80,22 @@ ProgressReporter <- R6::R6Class("ProgressReporter",
     },
 
     show_status = function(complete = FALSE) {
+      time <- proc.time()
       if (complete) {
         if (self$ctxt_n_fail > 0) {
           status <- crayon::red(cli::symbol$cross)
         } else {
           status <- crayon::green(cli::symbol$tick)
         }
       } else {
+
+        # Do not print if not enough time has passed since we last printed.
+        if ((time - self$last_time)[[3]] < self$min_time) {
+          return()
+        }
```

Comment:
I would always print a line at the beginning of a new context/file, so that you at least know which test file is running. I.e. this:
```
⠏ |  0       | Expectations
```

Otherwise, if the first test case takes very long, you'll have no output from the file for a long time.